input/pagecontent/profiles-and-extensions.md

```diff
@@ -15,9 +15,9 @@ The following profiles and have been defined for this implementation guide.
 
 ### Extensions
 
-All extensions used in this guide are defined in the base FHIR specification or [AU Base](http://build.fhir.org/ig/hl7au/au-fhir-base/profiles-and-extensions.html#extensions).
+All extensions used in this guide are defined in the [FHIR Extensions Pack](http://hl7.org/fhir/extensions/ImplementationGuide/hl7.fhir.uv.extensions) or [AU Base](http://build.fhir.org/ig/hl7au/au-fhir-base/profiles-and-extensions.html#extensions).
```

Comment:
Remove hyperlink please, means we don't have to maintain links in publications.

Reply:
Hyperlink removed.